lib/inferno/dsl/fhir_client.rb

```diff
@@ -72,12 +81,14 @@ def fhir_clients
       # @param headers [Hash] custom headers for this operation
       # @return [Inferno::Entities::Request]
       def fhir_operation(path, body: nil, client: :default, name: nil, headers: {})
-        store_request_and_refresh_token(fhir_client(client), name) do
-          operation_headers = fhir_client(client).fhir_headers
-          operation_headers.merge!('Content-Type' => 'application/fhir+json') if body.present?
-          operation_headers.merge!(headers) if headers.present?
+        fhir_error_filter do
```

Comment:
Ideally (though maybe its not practical), we would be able to document that we made the request and received no response, so it would show up in our interface.  By wrapping this outside of the store block, i assume we would still never store the request.  Maybe this would be a separate ticket though... I realize that this would be beyond the current scope.